api/endpoints/location_profiles.go

```diff
@@ -0,0 +1,83 @@
+package endpoints
```

Comment:
Again, isn't there a way to DRY on the repeated CRUD pattern?
E.g.location_profiles.go and monitor_groups are basically the same file, just different types.
Would like to see the CRUD pattern abstracted.